README.md

```diff
@@ -39,10 +40,20 @@ Create your new page using:
 Create your new post using:
 
     $ bundle exec jekyll post "My New Post"
+    # or by using the compose command
+    $ bundle exec jekyll compose "My New Post"
+    # or by using the compose command with post specified
+    $ bundle exec jekyll compose "My New Post" --post
+    # or by using the compose command with the posts collection specified
+    $ bundle exec jekyll compose "My New Post" --collection "posts"
```

Comment:
Since these commands are not part of a workflow, let us split this into separate individual examples. Also, using syntax-highlighting will further delineate comments:
```sh
    # or by using the compose command
    $ bundle exec jekyll compose "My New Post"
```
```sh
    # or by using the compose command with post specified
    $ bundle exec jekyll compose "My New Post" --post
```
```sh
    # or by using the compose command with the posts collection specified
    $ bundle exec jekyll compose "My New Post" --collection "posts"
```